tasks/src/zero-shot-classification/data.ts

```diff
@@ -2,7 +2,20 @@ import type { TaskDataCustom } from "../Types";
 
 const taskData: TaskDataCustom = {
 
-	datasets:     [],
+	datasets: [
+		{
+			description: "A widely used dataset used to benchmark multiple variants of text classification.",
+			id:          "glue",
+		},
+		{
+			description: "The Multi-Genre Natural Language Inference (MultiNLI) corpus is a crowd-sourced collection of 433k sentence pairs annotated with textual entailment information.",
+			id:          "MultiNLI",
```

Comment:
```suggestion
			id:          "multi_nli",
```

This is the dataset ID on HF Hub :)